docs/Getting_Involved/Newcomers.rst

```diff
@@ -0,0 +1,117 @@
+
+Now that you are part of our organization, you can start working on issues.
+If you are familiar with git, you can skip the next section and pick an issue.
+
+
+Get help with git
+-----------------
+
+We use github to manage our repository. If you're not familiar with git, we
+strongly recommend following a tutorial, such as `this one <https://try.github.io/levels/1/challenges/1>`_.
+
+If there's anything unclear, or you are encountering problems, feel free
+to contact us on gitter, and we will help you!
+
+Start working
+-------------
+
+Now is the time to pick an issue.
+Here are two links that will lead you to newcomers issues:
+
+- `Newcomer issues <https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+user%3Acoala-analyzer+label%3Adifficulty%2Fnewcomer>`_
+
+- For more information about what bears are, please check the following link: `Writing bears <http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html>`_
+
+The easy issues that will help you get started are labeled as
+"difficulty/newcomer" and are only there to give you a glimpse of how it is
+working with us.
+
+Now pick an issue which isn't assigned, and if you want to fix
+it, then leave a comment that you would like to get assigned. This way
+we don't have more people working on the same issue at the same time.
+Now you can start working on it.
+For more info on how to work correctly with git, try `this <https://github.com/coala-analyzer/coala/wiki/Getting-Started-to-Contribute#a-little-git-advice>`_.
+
+.. note::
+
+    Before starting to write your first commit, check out this link:
+    `Writing good commits <http://coala.readthedocs.org/en/latest/Getting_Involved/Writing_Good_Commits.html>`_
+
+Sending your changes
+--------------------
+
+Now that you've fixed the issue, you've tested it and you think it is ready
+to be used, create a commit and push it to your fork, using:
+
+::
+
+    $ git push origin fix-branch
+
+Creating a Pull Request
+-----------------------
+
+Now that your commit has been sent to your fork, it is time
+to do a Pull Request. It can be done by accessing your fork on github and
+clicking "New Pull Request".
+
+**Congratulations!** You have now created your first Pull Request!
+
+What to do after creating a PR
+------------------------------
+
+After creating your Pull Request, there's two options:
+- your Pull Request gets accepted, and your commit will get merged into the
+master branch;
+- your Pull Request doesn't, and therefore you will need to edit your code
+or your commit message;
+
+It's highly unlikely that your PR will be accepted on the first attempt - but
+don't worry that's just how it works. It helps us keeping coala clean and stable.
+
+.. note::
+    See also: `Review Process <http://coala.readthedocs.org/en/latest/Getting_Involved/Review.html>`_
+
+Most of the time it is the second option. Now if you need your code, you can
+simply edit it again, add it and commit it using
```

Comment:
"Most of the time it is the second option" was already mentioned before this reads weird